src/test/scala/nl/grons/metrics/scala/TimerSpec.scala

```diff
@@ -79,6 +79,18 @@ class TimerSpec extends FunSpec with OneInstancePerTest {
       verify(context).stop()
     }
 
+    it("should measure a future closure which errors") {
+      import ExecutionContext.Implicits.global
+      val error = new Exception
+      val caught = intercept[Exception] {
+        timer.timeFuture(throw error)
+      }
+      caught should equal (error)
```

Comment:
The advantage of checking it's the same object, is that you make sure the original stack trace is not meddled with.